doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+# Introduction
+
+The mission of the US Department of Energy's Nuclear Energy Advanced Modeling and Simulation ([NEAMS](https://neams.inl.gov/)) Program is to develop, apply, and deploy state-of-the-art predictive modeling and simulation tools for the design and analysis of current and future nuclear energy systems. NEAMS develops state-of-the-art [scalable tools](https://neams.inl.gov/code-descriptions/), such as the Bison fuel performance code, Griffin reactor physics code, System Analysis Module (SAM) code, and Nek5000 computational fluid dynamics code.
+[NEAMS tools](https://neams.inl.gov/code-descriptions/) are also designed to be interoperable, allowing users to construct customized multiphysics workflows. This flexibility is achieved by using the Multiphysics Object-Oriented Simulation Environment ([MOOSE](https://mooseframework.inl.gov/index.html)) framework, which is the foundation of several solvers and also provides the functionality to couple individual applications in loose or tight coupling schemes through the [MOOSE MultiApp System](https://mooseframework.inl.gov/syntax/MultiApps/index.html).
+
+[The NEAMS Workbench](https://www.ornl.gov/onramp/neams-workbench) [!citep](Lefebvre19)  initiative is designed to facilitate the transition from conventional design tools to more advanced modeling and simulation tools by providing a common user interface for model creation, review, execution, and visualization on personal computers and high-performance computing (HPC) platforms.
+
+The MOOSE MultiApp System allows users to assemble individual codes together to create a customized multiphysics simulation for any reactor design. To couple codes, a hierarchical workflow is invoked involving multiple input files linked together through input syntax that specifies execution order and data transfers between codes. Typically, the MOOSE-based super-application is invoked on an HPC platform. Examples of multiphysics problems specific to nuclear reactor applications are available in the Virtual Test Bed ([VTB](https://mooseframework.inl.gov/virtual_test_bed/)) repository.
```

Comment:
I would not have this paragraph here. I d focus on workbench

Reply:
I will remove it.

Reply:
Done.